src/views/VSettings.vue

```diff
@@ -74,6 +74,7 @@ const checkboxKeys: readonly (keyof PickByType<
 	"lookForTagsForTimeline",
 	"applyAdditonalConditionFormatting",
 	"stylizeDateInline",
+        "showNoteHeadersInCardBody",
```

Comment:
Same here, the tabs are not conform to the lint configuration try runnning
`npx eslint . --config .eslintrc --ext .js,.jsx,.ts,.tsx --fix`

Reply:
As above: will fix :-/